### PR TITLE
[API Compatibility No.186] Keep target for torch.nn.functional.mse_loss in PaConvert

### PR DESCRIPTION
**Pull request by Aidenwu0209**
### PR Category
User Experience

### PR Types
New features

### PR Docs
- Paddle PR: https://github.com/PaddlePaddle/Paddle/pull/78128
- Docs PR: https://github.com/PaddlePaddle/docs/pull/7840

### PR APIs
```bash
torch.nn.functional.mse_loss
```

### Description
- Remove `kwargs_change: {target: label}` from `torch.nn.functional.mse_loss` mapping.
- Keep `SizeAverageMatcher` behavior for `size_average/reduce/reduction`, while preserving PyTorch-style `target` in converted Paddle code.
- This aligns with Paddle side alias support: `paddle.nn.functional.mse_loss(target=...)`.

### CODE REVIEW

File: tests/test_nn_functional_mse_loss.py [hunk 1]
```diff
@@ -158,7 +158,16 @@ def test_case_10():
         result = torch.nn.functional.mse_loss(input=input, target=target, size_average=None, reduce=True, reduction='mean')
         """
     )
-    obj.run(pytorch_code, ["result"])
+    expect_paddle_code = textwrap.dedent(
```
Comment (zhwesky2010) [inline tests/test_nn_functional_mse_loss.py]: 这个要改成实际运行的形式来测，参考开发流程
